### ISSUE TITLE
Incorrect message from the Delegation module

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#545]
When there are too few IPv6 name servers the Delegation module creates the following  message referring to IPv4 addresses:
```
   4.96 ERROR     Child does not list enough (1) nameservers that resolve to IPv6 addresses (46.253.202.186;79.99.1.90;82.102.5.100). Lower limit set to 2.
```

The error was found on an installation based on the develop branch on commit 3d0c148 and was found in the following test:
```
# zonemaster-cli pearldragon.se --level info --no-ipv6 --test delegation
Seconds Level     Message
======= ========= =======
   0.00 INFO      Using version v3.0.2 of the Zonemaster engine.
   1.78 INFO      Parent lists enough (3) nameservers (kif.hildingsson.se;lakridstrollet.regexp.se;ns.narnia.pp.se). Lower limit set to 2.
   4.95 INFO      Child lists enough (3) nameservers (kif.hildingsson.se;lakridstrollet.regexp.se;ns.narnia.pp.se). Lower limit set to 2.
   4.96 INFO      Child lists enough (3) nameservers that resolve to IPv4 addresses (46.253.202.186;79.99.1.90;82.102.5.100). Lower limit set to 2.
   4.96 ERROR     Child does not list enough (1) nameservers that resolve to IPv6 addresses (46.253.202.186;79.99.1.90;82.102.5.100). Lower limit set to 2.
   5.43 INFO      Delegation lists enough (3) nameservers that resolve to IPv4 addresses (46.253.202.186;79.99.1.90;82.102.5.100). Lower limit set to 2.
   5.43 ERROR     Delegation does not list enough (1) nameservers that resolve to IPv6 addresses (2a02:750:7::4c6). Lower limit set to 2.
(...)
```

**Comment by vlevigneron** [comment]
The fix has already been merged.

**Comment by matsduf** [comment]
Fixed by #552.
